src/GraphQL.NewtonsoftJson/DocumentWriter.cs

```diff
@@ -9,49 +9,87 @@
 
 namespace GraphQL.NewtonsoftJson
 {
+    /// <summary>
+    /// Serializes an ExecutionResult to a stream using the Newtonsoft.Json library.
```

Comment:
The same comment about ExecutionResult here

Reply:
All of the NewtonsoftJson constructor comments are identical to System.Text.Json comments.  I will change them in tandem

Reply:
OK